src/rules/convertRules.test.ts

```diff
@@ -5,6 +5,19 @@ import { RuleConverter, ConversionResult } from "./converter";
 import { RuleMerger } from "./merger";
 
 describe("convertRules", () => {
+    it("doesn't crash with a type error when passed undefined configuration", () => {
```

Comment:
Just nitpicking here: technically a type error wouldn't do anything at runtime.

```suggestion
    it("doesn't crash when passed an undefined configuration", () => {
```

Reply:
![image](https://user-images.githubusercontent.com/84951/83448878-a1e0e280-a420-11ea-86de-d3573b73c370.png)

Reply:
Oh haha good point. You're right. 

I think the gist of the suggestion is correct for a different reason though: in general, other unit tests don't mention the particular error type they might crash with, just that the action succeeds. The naming here confused me.

This is definitely a small thing and non-blocking. I'd be happy to merge if you disagree and do nothing. 😄

Reply:
I'll make the change; the test case is more general than my particular text suggests.